paddle/fluid/platform/gpu_info.cc

```diff
@@ -557,6 +558,7 @@ class RecordedCudaMallocHelper {
 #ifdef PADDLE_WITH_HIP
     auto result = hipMalloc(ptr, size);
 #else
+    CUDAGraphCaptureModeGuard capture_mode_guard{cudaStreamCaptureModeRelaxed};
```

Comment:
imho, call this func when IsCUDAGraphCapturing is true.

Reply:
Done.